client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks/use-processing-loading-messages.ts

```diff
@@ -1,11 +1,13 @@
 import { Onboard } from '@automattic/data-stores';
+import { isWooExpressFlow } from '@automattic/onboarding';
 import { useSelect } from '@wordpress/data';
 import { useI18n } from '@wordpress/react-i18n';
 import { STEPPER_INTERNAL_STORE } from 'calypso/landing/stepper/stores';
+import { LoadingMessage } from './types';
```

Comment:
Nit: this should be `import type`.
```suggestion
import type { LoadingMessage } from './types';
```